packages/types/src/scripts/METHODS_STORAGE_SUBSTRATE.md

```diff
@@ -14,6 +14,9 @@ _These are keys that are always available to the runtime implementation_
 ▸ **changesTrieConfig**(): `u32`
 - **summary**: Changes trie configuration is stored under this key.
 
+▸ **childStorageKeyPrefix**(): `u32`
```

Comment:
Remove, auto-generated, done by CI.

Reply:
This is not auto generated, it generates `docs/METHODS_STORAGE.md ` according to your comment here https://github.com/polkadot-js/api/pull/1030#discussion_r294729850 .

Reply:
Indeed. I get very jittery with all manual edits to `*.md` files, so misread where this one was from after a barrage elsewhere :)